src/tests/modules/eap_sim/sim_xlat_id_aka.unlang

```diff
@@ -98,12 +86,8 @@ if (&control.Tmp-String-1 != "0%{User-Name[1]}") {
 #
 #  2.1 - Encrypt the permanent ID for fastauth
 #
-update control {
-	&Tmp-String-0 := '1420032219455259'
-}
-update control {
-	&User-Name[1] := "%(3gpp_temporary_id_encrypt:%{User-Name[1]} %{control.Tmp-String-0} 6 fastauth)"
-}
+&control.Tmp-String-0 := '1420032219455259'
+&control.User-Name[1] = "%(3gpp_temporary_id_encrypt:%{User-Name[1]} %{control.Tmp-String-0} 6 fastauth)"
```

Comment:
That should be fixed to append the `User-Name` to the `control` list.  As discussed in chat, assigning *new* values to index `[1]` is almost always wrong.